crates/web5/src/dids/resolution/resolution_result.rs

```diff
@@ -40,6 +40,13 @@ impl ResolutionResult {
             },
         }
     }
+
+    pub(crate) fn from_error(error: ResolutionMetadataError) -> Self {
```

Comment:
Slight preference for `From<ResolutionMetadataError> for ResolutionResult`

Reply:
done